test/org/opensha/commons/calc/magScalingRelations/magScalingRelImpl/MagScalingRelSuite.java

```diff
@@ -0,0 +1,13 @@
+package org.opensha.commons.calc.magScalingRelations.magScalingRelImpl;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+import org.opensha.commons.calc.*;
+
+@RunWith(Suite.class)
+@Suite.SuiteClasses({
+        TestTMG2017_MagAreaRel.class
+})
+
+public class MagScalingRelSuite {
```

Comment:
I can run all tests just fine in IntelliJ. But you could add this method to the class, maybe that will help you run the tests
```
	public static void main(String args[])
	{
		org.junit.runner.JUnitCore.runClasses(MagScalingRelSuite .class);
	}
```

Reply:
Thanks for confirming that the tests works. I could now get it working here (on Eclipse) as well - the fix is:
> run as -> run configurations -> dependencies -> classpath entries -> advanced -> add library -> Junit.